mantidproject/mantid
# Powder ISIS script fails when ExistingV is no

fixes #14433 

> - [x] merge #14426 before this

**To Test**
- Run `ISISPowderDiffractionHrpdTest.py` system test locally
- Check all tests pass
- Once the system test has passed, make sure the value of `ExistingV` in the pref file `mtd2.pref` hasn't changed to `yes` from `no`
- You could also run the python script manually via Mantid using the system test script which is utilising  `mtd2.pref` preference file. 
- New parameter `Write_ExtV` has been included for `cry_focus.focus_all(..)` with default value of `True`, this will enable the user to disable the script to auto replace `ExistingV` value in pref file.
